examples/with-okta-multi-tenant-pkce-flow/src/index.tsx

```diff
@@ -0,0 +1,105 @@
+import ReactDOM from 'react-dom/client';
+import {
+  BrowserRouter,
+  Routes,
+  Route,
+} from "react-router-dom";
+import './index.css';
+import App from './App';
+import * as reactRouterDom from "react-router-dom";
+
+import SuperTokens, { SuperTokensWrapper, getSuperTokensRoutesForReactRouterDom } from "supertokens-auth-react";
+import ThirdParty from "supertokens-auth-react/recipe/thirdparty";
+
+import Session from "supertokens-auth-react/recipe/session";
+
+import Index from "./routes/index"
+import Dashboard from "./routes/dashboard"
+import { PreAndPostAPIHookAction } from 'supertokens-web-js/recipe/thirdparty';
+
+SuperTokens.init({
+    appInfo: {
+        appName: "Multitenant Example",
+        apiDomain: "http://multitenant.com:8000",
+        websiteDomain: window.location.origin,
+    },
+    recipeList: [
+        ThirdParty.init({
+            signInAndUpFeature: {
+                providers: [
+                  {
+                    id: "okta",
+                    name: "Okta",
+                    buttonComponent: <div style={{
+                        cursor: "pointer",
+                        border: "1",
+                        paddingTop: "5px",
+                        paddingBottom: "5px",
+                        borderRadius: "5px",
+                        borderStyle: "solid"
+                    }}>Login with Okta</div>
+                  }
+                ]
+            },
+            getRedirectionURL: async (context) => {
+              if (context.action === "SUCCESS") {
+                  if (context.redirectToPath !== undefined) {
+                      // we are navigating back to where the user was before they authenticated
+                      return context.redirectToPath;
+                  }
+                  return "/dashboard";
+              }
+              return undefined;
```

Comment:
Explain why this is there -> so that post login, the user is redirected to the dashboard

Reply:
done